src/components/Signup/Signup.js

```diff
@@ -56,60 +61,79 @@ function Signup() {
-          onChange={(event) => setValues((prev) => ({ ...prev, email: event.target.value }))}
-        />
-        <InputControl
-          label='Password'
-          type='password'
-          placeholder='Enter Password'
-          onChange={(event) => setValues((prev) => ({ ...prev, pass: event.target.value }))}
-        />
-        <br></br>
-        {/* TODO: I would suggest we make it select input */}
-        <InputControl
-          label='Gender'
-          placeholder='Enter your gender'
-          onChange={(event) => setValues((prev) => ({ ...prev, gender: event.target.value }))}
-        />
-        {/* TODO: I would suggest we make it calendar input */}
-        <InputControl
-          label='Birth date'
-          placeholder='Enter your birth date'
-          onChange={(event) => setValues((prev) => ({ ...prev, date: event.target.value }))}
-        />
-        {/* TODO: please add a placeholder */}
-        <InputControl
-          label='Height'
-          placeholder='Enter your height '
-          onChange={(event) => setValues((prev) => ({ ...prev, height: event.target.value }))}
-        />
-        {/* TODO: please add a placeholder */}
-        <InputControl
-          label='Weight'
-          placeholder='Enter your weight'
-          onChange={(event) => setValues((prev) => ({ ...prev, weight: event.target.value }))}
-        />
-        <div className={styles.footer}>
-          <b className={styles.error}>{errorMsg}</b>
-          <button onClick={handleSubmission} disabled={submitButtonDisabled}>
-            Sign up
-          </button>
-          <p>
-            Already have an account?{' '}
-            <span>
+    <>
+      <NavBar />
+      <Form className={styles.container}>
+        <div className={styles.innerBox}>
+          <h1 className={styles.heading}>Sign up</h1>
+          <Form.Group>
+            <Form.Label>Email</Form.Label>
+            <Form.Control
+              placeholder='Enter email address'
+              onChange={(event) => setValues((prev) => ({ ...prev, email: event.target.value }))}>
+            </Form.Control>
+          </Form.Group>
+          <Form.Group>
+            <Form.Label>Password</Form.Label>
+            <Form.Control
+              type='password'
+              placeholder='Enter Password'
+              onChange={(event) => setValues((prev) => ({ ...prev, pass: event.target.value }))}>
+            </Form.Control>
+          </Form.Group>
+          <br></br>
+          <Form.Group>
+            <Form.Label>Gender</Form.Label>
+            <Form.Select
+              placeholder='Choose your gender'
+              onChange={(event) => setValues((prev) => ({ ...prev, gender: event.target.value }))}>
+              {genders.map(option => (
+                <option key={option}>{option}</option>
+              ))}
+            </Form.Select>
+          </Form.Group>
+          <Form.Group>
+            <Form.Label>Birth date</Form.Label>
```

Comment:
Can you change it to `Date of birth` please

Reply:
Changed